velox/vector/tests/utils/VectorTestBase.cpp

```diff
@@ -127,6 +129,23 @@ void assertEqualVectors(
   });
 }
 
+void assertVectorHashEqualsIgnoreOrder(
+    const VectorPtr& expected,
+    const VectorPtr& actual) {
+  ASSERT_EQ(expected->size(), actual->size());
+  ASSERT_TRUE(expected->type()->equivalent(*actual->type()))
+      << "Expected " << expected->type()->toString() << ", but got "
+      << actual->type()->toString();
+
+  std::multiset<uint64_t> expectedSet;
+  std::multiset<uint64_t> actualSet;
+  for (auto i = 0; i < expected->size(); i++) {
+    expectedSet.insert(expected->hashValueAt(i));
+    actualSet.insert(actual->hashValueAt(i));
```

Comment:
Added a new test utility function `assertVectorHashEqualsIgnoreOrder()` to compare all hash values ignoring the element order in the vector. This is super useful for skipping-order two vector comparison.
My original plan is to call functions like `valueAt()`, but it is not available in `BaseVector` class.